lib/active_merchant/billing/gateways/xpay.rb

```diff
@@ -206,17 +201,17 @@ def parse(body = {})
 
       def commit(action, params, options)
         transaction_id = transaction_id_from(params, options, action)
-        begin
```

Comment:
⚠️ Warning:

Leaving a comment here for the `parse` method, if we are going to provide a default value for it should be `"{}"` not `{}`

Reply:
I will remove it, instead of that I will ensure that we will parse a valid json